lib/myfinance/resources/receivable_account.rb

```diff
@@ -8,12 +8,19 @@ module Resources
     #
     class ReceivableAccount < FinancialAccount
 
+      def receive(id, entity_id, params = {})
+        http.put(endpoints(id, entity_id, :receive), body: { receivable_account: params }) do |response|
+          respond_with_object(response, 'receivable_account')
+        end
+      end
+
       private
 
-      def endpoints
+      def endpoints(id, entity_id, key)
         {
-          create: '/entities/:entity_id/receivable_accounts'
-        }
+          create: '/entities/:entity_id/receivable_accounts',
+          receive: '/entities/:entity_id/receivable_accounts/:id/receive'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.